Framework/CMakeLists.txt

```diff
@@ -139,29 +139,53 @@ set(
 
 set(
   EXE_NAMES
-  qcInfoService
-  qcInfoServiceDump
-  qcRunBasic
-  qcRunAdvanced
-  qcRunReadout
-  runMergerTest
-  qcRunReadoutForDataDump
-  repositoryBenchmark
+  o2-qc-info-service
+  o2-qc-info-service-dump
+  o2-qc-run-basic
+  o2-qc-run-advanced
+  o2-qc-run-readout
+  o2-qc-run-merger-test
+  o2-qc-run-readout-for-data-dump
+  o2-qc-repository-benchmark
 )
 
+# These were the original names before the convention changed.
+# We will get rid of them but for the time being we want to create symlinks to avoid confusion.
+set(
+        EXE_OLD_NAMES
```

Comment:
Could you modify it to have the two-space indentation, as in the rest of the file?